### ISSUE TITLE
Checksum mismatch when running `go mod tidy`

### ISSUE THREAD

**Issue by erichsueh3** [moricho/tparallel#22]
```
go: downloading github.com/moricho/tparallel v0.3.0
go: upgraded github.com/moricho/tparallel v0.2.1 => v0.3.0: error finding sum for github.com/moricho/tparallel@v0.3.0: github.com/moricho/tparallel@v0.3.0: verifying module: checksum mismatch
	downloaded: h1:EthKzLoZA6C11YB1vMPWLYmF2nnPCUX712TjDgsP31A=
	sum.golang.org: h1:8dDx3S3e+jA+xiQXC7O3dvfRTe/J+FYlTDDW01Y7z/Q=
```

This issue arises when running `go mod tidy` on a project that uses this package. We suspect that this is happening because a tag has been overwritten, instead of a new patch version being published for a major/minor version. Can we get confirmation on whether or not this is the case, and if we can have a path to resolution on this? Thanks!

**Comment by ldez** [comment]
Hello @moricho, I'm a golangci-lint core maintainer, do you plan to create a new tag?

**Comment by moricho** [comment]
@erichsueh3 
Thanks for reporting the issue. I overwrote the `v0.3.0` tag as you said and that might cause this...

@ldez @erichsueh3 
I've released a new version [`v0.3.1`](https://github.com/moricho/tparallel/releases/tag/v0.3.1) 🚀

**Comment by ldez** [comment]
Thank you @moricho :+1: 

tparallel has been updated in golangci-lint.

https://github.com/golangci/golangci-lint/pull/3719
